site/content/en/docs/Guides/feature-stages.md

```diff
@@ -40,7 +40,7 @@ The current set of `alpha` and `beta` feature gates:
 | Example Gate (not in use)                                                                                                   | `Example`                          | Disabled | None    | 0.13.0 |
 
 {{% /feature %}}
-
+| [Scheduled Fleet Autoscaling](https://github.com/googleforgames/agones/issues/3008)                                         | `ScheduledAutoscaler`              | Disabled | `Alpha` | 1.43.0 |
```

Comment:
Apologies for not being clearer on my previous comment, this should go down on line 57. The code block that will be published with version 1.43 starts with `{{% feature publishVersion="1.43.0" %}}` on line 44 and is closed by `{{% /feature %}}` on line 59.

Reply:
Oh I see, np.